met-web/src/components/engagement/old-view/widgets/WhoIsListeningWidget.tsx

```diff
@@ -38,33 +47,106 @@ const WhoIsListeningWidget = ({ widget }: WhoIsListeningWidgetProps) => {
             dispatch(
                 openNotification({
                     severity: 'error',
-                    text: 'Error occurred while fetching Engagement wdigets information',
+                    text: 'Error occurred while fetching engagement widget information',
+                }),
+            );
+        }
+    };
+
+    const getListeningWidget = async () => {
+        try {
+            const listeningWidget = await fetchListeningWidget(widget.id);
+            setListeningWidget(listeningWidget);
+            setIsLoading(false);
+        } catch (error) {
+            setIsLoading(false);
+            console.log(error);
```

Comment:
Even though we may retire this code at some point soon, I'd still recommend you use `console.error` and give the log a descriptive label to help us locate it in the code if we're troubleshooting.
```suggestion
            console.error('Error loading who is listening widget.', error);
```

Reply:
Yes sorry, that was just for my benefit, it's been removed.